CHANGELOG.md

```diff
@@ -3,6 +3,123 @@
+  +apiRef.current.setCellFocus: (id: GridRowId, field: string) => void;
+  ```
+
+- [XGrid] Rename `apiRef` methods (#1513) @dtassone
+
+  Changes on `apiRef.current`:
+
+  ```diff
+  -apiRef.current.getRowIndexFromId: (id: GridRowId) => number;
+  +apiRef.current.getRowIndex: (id: GridRowId) => number;
+  ```
+
+- [XGrid] Rename `apiRef` methods (#1667) @m4theushw
+
+  Changes on `apiRef.current`:
+
+  ```diff
+  -apiRef.current.getColumnFromField: (field: string) => GridColDef;
+  -apiRef.current.getRowFromId: (id: GridRowId) => GridRowModel;
+  +apiRef.current.getColumn: (field: string) => GridColDef;
+  +apiRef.current.getRow: (id: GridRowId) => GridRowModel;
+  ```
+
+#### Changes
+
+- [DataGrid] Add Slovak (skSK) locale (#1634) @martinvysnovsky
+- [DataGrid] Add `columnHeader`, `row` and `cell` in addition to `root` in classes prop (#1660) @DanailH
+- [DataGrid] Add `isRowSelectable` prop (#1659) @m4theushw
+- [DataGrid] Add sort icon for when column is unsorted (#1415) @m4theushw
+- [DataGrid] Fix `id` and `aria-labelledby` attributes on the column menu (#1460) @m4theushw
+- [DataGrid] Fix broken checkbox in Material-UI v5 (#1587) @ZeeshanTamboli
+- [DataGrid] Fix CSS classes prefix (#1693) @m4theushw
+- [DataGrid] Fix German (deDe) locale (#1624) @klinge27
+- [DataGrid] Fix filter with object as value and value getter (#1665) @dtassone
+- [DataGrid] Fix incorrect date selection (#1652) @aTmb405
+- [DataGrid] Fix overflow of maximum page (#1583) @oliviertassinari
+- [DataGrid] Fix typo in Italian (itIT) locale (#1635) @profcav
+- [DataGrid] Improve performance of width resizing (#1686) @dtassone
+- [DataGrid] Make rows immutable for better developer experience (#1661) @ZeeshanTamboli
+- [DataGrid] Pass state values as props (#1628) @m4theushw
+- [DataGrid] Improve performance with filtering, sorting, and rendering (#1513) @dtassone
+- [XGrid] Fix checkbox column resizing (#1682) @elyesbenabdelkader
+
+### Docs
+
+- [docs] Add description for all events (#1572) @m4theushw
+- [docs] Add missing CSS rules (#1694) @ZeeshanTamboli
+- [docs] Add missing descriptions in `GridFilterApi` (#1620) @m4theushw
+- [docs] Clean demos components (#1681) @oliviertassinari
+- [docs] Fix docs demo (#1691) @dtassone
+- [docs] Improve Filtering page (#1671) @m4theushw
+- [docs] Improve the data grid components page (#1382) @dtassone
+- [docs] Refine the descriptions to be clearer (#1589) @oliviertassinari
+- [docs] Reshuffle columns and rows styling sections (#1622) @DanailH
+
+### Core
+
+- [core] Fix dependabot config (#1619) @oliviertassinari
+- [core] Remove `makeStyles` dependency on `@material-ui/core/styles` (#1627) @mnajdova
+- [core] Remove `withStyles` dependency on `@material-ui/core/styles` (#1690) @mnajdova
+- [core] Replace `classnames` utility with `clsx` dependency (#1586) @ZeeshanTamboli
+- [core] Reuse `colIndex` already computed (#1666) @oliviertassinari
+- [test] Add constraints on cell render (#1662) @oliviertassinari
```

Comment:
I guess `tests` fall under `core` so we can change them to `core`?

Reply:
Listing test PRs inside "Core" is a convention from the core. I was following it.

<img width="600" src="https://user-images.githubusercontent.com/42154031/118708828-971a8b80-b7f2-11eb-90ef-2be28d9320ba.png" alt="image" style="max-width:100%;">